integration-cli/docker_cli_build_test.go

```diff
@@ -6223,3 +6223,22 @@ func (s *DockerSuite) TestBuildNoNamedVolume(c *check.C) {
 	_, err := buildImage("test", dockerFile, false)
 	c.Assert(err, check.NotNil, check.Commentf("image build should have failed"))
 }
+
+// #15780
+func (s *DockerSuite) TestBuildMultipleTags(c *check.C) {
+	dockerfile := `
+	FROM busybox
+	MAINTAINER test-15780
+	`
+	cmd := exec.Command(dockerBinary, "build", "-t", "tag1", "-t", "tag2:v2",
+		"-t", "tag1:latest", "-t", "tag1", "--no-cache", "-")
```

Comment:
ah lol.. I think I need glasses (or sleep :sweat:), didn't see the diff :sweat_smile:. 
This makes sense for me yeah, :stuck_out_tongue_closed_eyes:.